lib/render/editor/editor.html.erb

```diff
@@ -233,6 +230,11 @@
       },
 
       _relocateTrashOnResize() {
+        if (this.readOnly) {
+          console.debug("No trash relocate on resize required");
+          return;
```

Comment:
This will be a problem if, for any reason, `readOnly` constant turns into a variable in the future. Perhaps we can always subscribe to the event, the actual relocation won't happen while the editor is read only.